<launch>
	<!-- launch ros core and world setup -->
	<arg name="world" default="thin_beer"/>
	<include file="$(find pr2_grasping)/launch/world.launch">
		<arg name="world" value="$(arg world)"/>
	</include>

	<!-- launch the robot -->
	<include file="$(find pr2_grasping)/launch/pr2.launch"/>

	<rosparam param="/move_group/max_range">5.0</rosparam>
	<rosparam param="/move_group/octomap_resolution">0.03</rosparam>  <!-- 0.025 -->
</launch>